ivy/array/experimental/creation.py

```diff
@@ -54,3 +54,38 @@ def eye_like(
                     0., 1., 0.]])
         """
         return ivy.eye_like(self._data, k=k, dtype=dtype, device=device, out=out)
+    
+    
+    def sequence_empty(
```

Comment:
Sorry! Just one more thing. I noticed there are some lint errors related to your changes. Here they are: 
```
ivy/array/experimental/creation.py:59:5: E303 too many blank lines (2)
ivy/array/experimental/creation.py:6:89: E501 line too long (89 > 88 characters)
ivy/container/experimental/creation.py:819:5: E303 too many blank lines (2)
ivy/container/experimental/creation.py:845:5: E303 too many blank lines (2)
ivy/container/experimental/creation.py:867:1: W391 blank line at end of file
```
Don't worry about the others. Did you install pre-commit?